DirectProgramming/DPC++FPGA/Tutorials/Features/read_only_cache/README.md

```diff
@@ -0,0 +1,261 @@
+     ```
+     make fpga
+     ``` 
+3. (Optional) As the above hardware compile may take several hours to complete,
+   FPGA precompiled binaries (compatible with Linux* Ubuntu* 18.04) can be
+   downloaded <a
+   href="https://iotdk.intel.com/fpga-precompiled-binaries/latest/read_only_cache.fpga.tar.gz"
+   download>here</a>.
+
+### On a Windows* System
+
+1. Generate the `Makefile` by running `cmake`.
+     ```
+   mkdir build
+   cd build
+   ```
+   To compile for the Intel® PAC with Intel Arria® 10 GX FPGA, run `cmake`
+   using the command:  
+    ```
+    cmake -G "NMake Makefiles" ..
+   ```
+   Alternatively, to compile for the Intel® FPGA PAC D5005 (with Intel Stratix®
+   10 SX), run `cmake` using the command:
+
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=intel_s10sx_pac:pac_s10
+   ```
+   You can also compile for a custom FPGA platform. Ensure that the board
+   support package is installed on your system. Then run `cmake` using the
+   command:
+   ```
+   cmake -G "NMake Makefiles" .. -DFPGA_BOARD=<board-support-package>:<board-variant>
+   ```
+
+2. Compile the design through the generated `Makefile`. The following build
+   targets are provided, matching the recommended development flow:
+
+   * Compile for emulation (fast compile time, targets emulated FPGA device): 
+     ```
+     nmake fpga_emu
+     ```
+   * Generate the optimization report: 
+     ```
+     nmake report
+     ``` 
+   * Compile for FPGA hardware (longer compile time, targets FPGA device):
+     ```
+     nmake fpga
+     ``` 
+
+*Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005
+(with Intel Stratix® 10 SX) do not support Windows*. Compiling to FPGA hardware
+on Windows* requires a third-party or custom Board Support Package (BSP) with
+Windows* support.
+ 
+### In Third-Party Integrated Development Environments (IDEs)
+
+You can compile and run this tutorial in the Eclipse* IDE (in Linux*) and the
+Visual Studio* IDE (in Windows*). For instructions, refer to the following
+link: [Intel® oneAPI DPC++ FPGA Workflows on Third-Party
+IDEs](https://software.intel.com/en-us/articles/intel-oneapi-dpcpp-fpga-workflow-on-ide)
+
+
+## Examining the Reports
+Locate the pair of `report.html` files in the
+`read_only_cache_disabled_report.prj` and `read_only_cache_enabled_report.prj`
+directories. Open the reports in any of Chrome*, Firefox*, Edge*, or Internet
+Explorer*. Navigate to the "Area Analysis of System" section of each report
+(Area Analysis > Area Analysis of System) and expand the "Kernel System" entry
+in the table. Notice that when the read-only cache is enabled, a new entry
+shows up in the table called "Constant cache interconnect" indicating that the
+cache has been created.
+
+
+## Running the Sample
+
+ 1. Run the sample on the FPGA emulator (the kernel executes on the CPU):
+     ```
+     ./read_only_cache.fpga_emu     (Linux)
+     read_only_cache.fpga_emu.exe   (Windows)
```

Comment:
I think you should explain that the cache is not implemented in the emulator, so there is only a single executatable for the fpga_emu target, unlike the fpga and reports targets.